app/javascript/components/app.jsx

```diff
@@ -11,10 +11,25 @@ import { persistCache } from 'apollo-cache-persist';
 import WidgetController from '@components/widget-controller';
 import versionCompare from '@components/version-compare';
 import helioSchema from '@javascript/schema.json';
+import { onError } from 'apollo-link-error';
 import GlobalStyle from '../styles';
 
 let link = null;
 
+// eslint-disable-next-line new-cap
+const errorLink = new onError(({ graphQLErrors, networkError }) => {
+  if (graphQLErrors)
+    graphQLErrors.map(({ message, locations, path }) =>
+      // eslint-disable-next-line no-console
+      console.log(
+        // eslint-disable-line
+        `[GraphQL Error]: Message: ${message}, Location: ${locations}, Path: ${path}`,
```

Comment:
sorry, hard to to tell I was referring to line 25

Reply:
Both eslint comments were taken out, and I believe I did something wonky with `npm add` instead of `yarn add` so the entire yarn.lock file got restructured for some reason. It's the way it should be now with the apollo-link-error dependency added.